### DISCUSSION TITLE
Searching for a Foooocus Expert

### DISCUSSION THREAD

**Discussion by Lupo1312** [lllyasviel/Fooocus#1973] [Q&A]
Hello everyone,

My name is Tiziano, and I am an enthusiastic user of the Focus application on Github. I am keenly interested in becoming an expert in using this application, particularly in understanding how it generates images and becoming skilled at consistently producing a specific type of image.

I am looking for a professional with in-depth experience in Focus to provide personalized training. I am willing to pay for private lessons or a structured course, depending on what is most effective.

My specific areas of interest include:
- In-depth understanding of Focus image generation mechanisms.
- Developing skills to consistently generate a specific type of image with high quality.
- Best practices and tips for advanced usage of the app.

If you are someone with the experience and skills I am looking for, or if you know someone who might be interested, please contact me here on Github.

Thank you for your time, and I look forward to a productive collaboration!

Best regards,
Tiziano

---

**Comment by Phrixus2023** [comment]
here，https://www.youtube.com/@phrixusli/featured

**Reply by Lupo1312** [reply]
thanks sir, will watch immediately

**Reply by Lupo1312** [reply]
> here，https://www.youtube.com/@phrixusli/featured

Unfortunately no English translation sir
